### PR TITLE
P0 product listing page

### PR DESCRIPTION
**Pull request by sumitk-singh**
**Kindly review the PR of product list page.**

### CODE REVIEW

**Review by sanu29 — APPROVED** [review]
Good to merge

File: pages-css/compo-css/card.css [hunk 1]
```diff
@@ -5,6 +5,7 @@
     --color-light-gray: #ebeae8;
     --color-light-black: #7a7a7a;
     --color-dark: #000000;
+    --color-green: #00a62c;
```
Comment (srishtigautam21) [inline pages-css/compo-css/card.css:8]: Use var for defining colors

Comment (srishtigautam21) [inline pages-css/compo-css/card.css:8]: define all the colors in one common css file
edit:well you can do these later on

File: pages-css/products-page-css/layout-handeler.css [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+
+/* *** */
+
+
+/* section-1 */
+
+.home-page-content-1>p {
+    font-size: 15px;
+    padding-top: 10px;
```
Comment (srishtigautam21) [inline pages-css/products-page-css/layout-handeler.css:104]: Use rem here

File: pages-css/products-page-css/page-style.css [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+/* color variables */
+
+:root {
+    --color-orange: #ffb95e;
+    --color-light-orange: #ffe3a6;
+    --color-white: #ffffff;
+    --color-light-gray: #e8e8e8;
+    --color-light-black: #575757;
+    --color-light-red: #bf9393;
+}
```
Comment (srishtigautam21) [inline pages-css/products-page-css/page-style.css:18]: combine all the imports and colors in one css file

**Review by srishtigautam21 — COMMENTED** [review]
there are few small changes required which i have suggested

File: pages-css/products-page-css/layout-handeler.css (hunk 2 above)
Comment (sumitk-singh) [reply pages-css/products-page-css/layout-handeler.css:104]: Someone earlier suggested use px below 1 rem, so

Comment (srishtigautam21) [reply pages-css/products-page-css/layout-handeler.css:104]: ohh so you haven't changed the root font size to 10px.Then i think its fine

**Pull request merged**